ci/forms.py

```diff
@@ -136,3 +138,27 @@ class SubscriptionFilterForm(forms.Form):
     completed = forms.ChoiceField(
         choices=COMPLETED_CHOICES, required=False
     )
+
+
+class ReportGenerationForm(forms.Form):
+    output_file = forms.CharField(
+        widget=forms.TextInput(attrs={'placeholder': 'Name of output file'}),
+        required=True)
```

Comment:
If the user supplies the same file name for 2 different sets of parameters, would that cause problems on the report generation side?